### ISSUE TITLE
Markdown should be used only as intended

### ISSUE THREAD

**Issue by Jenneh** [Iota-School/notebooks-for-all#23]
## Problem and context

In some cases, markdown is used for visual differentiation. This is confusing when using screen readers because markdown reads to mean a specific thing. Markdown should be used only as intended. ie. code blocks can’t be used for emphasis, just code).  (ie. semantic HTML

## Possible solutions

Style guide.

## Tasks to complete

- [ ] Create a style rule: Don’t use code block styling in markdown cells just for styling

**Comment by tonyfast** [comment]
is there a best practices document we could get started to capture this knowledge?
there some a good articles out there about using markdown correctly and i think could borrow from them.

* https://www.smashingmagazine.com/2021/09/improving-accessibility-of-markdown/
* https://tidbits.com/2013/06/18/with-markdown-even-the-blind-can-write/
* https://developers.google.com/style/accessibility

@Jenneh do you have an example of code being used improperly the way it was described?

**Comment by tonyfast** [comment]
something we could try is including AXE directly in the application to warn folks of accessibility violations.
